### ISSUE TITLE
Very unpopular torrent

### ISSUE THREAD

**Issue by qstokkink** [Tribler/tribler#3223]
I got this today while downloading with the Windows x64 `next` build (this does not happen when running from source):
![afbeelding](https://user-images.githubusercontent.com/3630389/32775597-2fd4bf86-c930-11e7-81d3-88aa0918fb34.png)
![afbeelding](https://user-images.githubusercontent.com/3630389/32775714-7dc8e618-c930-11e7-91a6-c261916eb3de.png)
That's not very many seeders. Also, this download doesn't seem to progress now (it downloaded just fine when running from source).

Here is the `/downloads` GET result:
```javascript
{
   "downloads":[
      {
         "files":[
            {
               "included":true,
               "index":0,
               "progress":0.0,
               "name":"bbb_sunflower_1080p_60fps_stereo_abl.mp4",
               "size":514832018
            }
         ],
         "max_download_speed":0,
         "vod_prebuffering_progress_consec":0.0,
         "num_peers":0,
         "availability":0,
         "size":514832018,
         "speed_down":0.0,
         "speed_up":0.0,
         "ratio":0.0,
         "hops":0,
         "destination":"bla",
         "anon_download":false,
         "max_upload_speed":0,
         "progress":0.0,
         "time_added":1510656474,
         "status":"DLSTATUS_DOWNLOADING",
         "total_down":266825,
         "total_up":0,
         "num_seeds":-65131,
         "name":"bbb_sunflower_1080p_60fps_stereo_abl.mp4",
         "trackers":[
            {
               "url":"udp://tracker.publicbt.com:80/announce",
               "status":"Not working",
               "peers":0
            },
            {
               "url":"[DHT]",
               "status":"Working",
               "peers":0
            },
            {
               "url":"udp://tracker.openbittorrent.com:80/announce",
               "status":"Not working",
               "peers":0
            },
            {
               "url":"[PeX]",
               "status":"Working",
               "peers":0
            }
         ],
         "total_pieces":982,
         "vod_mode":false,
         "safe_seeding":false,
         "eta":514832018000000.0,
         "error":"",
         "vod_prebuffering_progress":0.0,
         "infohash":"674d163d2184353ce21f3de5196b0a6d7c2f9fc2"
      }
   ]
}
```

**Comment by qstokkink** [comment]
I figured out that the `bla` destination is the default on the Windows build.
![afbeelding](https://user-images.githubusercontent.com/3630389/32777213-92fa045e-c935-11e7-8c00-98968995af32.png)

**Comment by qstokkink** [comment]
For some reason after Tribler crashed, this stopped occurring.

**Comment by qstokkink** [comment]
I know how to reproduce this now (on the Windows x64 next build):

- Add a torrent and download it completely
- Remove this torrent (only the torrent, leave the data)
- Add the same torrent again, but give it a non-existent download location

You should now see -65k seeders.

**Comment by qstokkink** [comment]
This, and several other issues related to it, will be fixed once #3253 is merged.
